project/static/src/js/admin/select-list.js

```diff
@@ -0,0 +1,78 @@
+var React = require('react');
+var Select = require('react-select');
+
+
+var SelectRow = React.createClass({
+    displayName: "SelectRow",
+
+    render: function () {
+        return React.createElement("div", {
+                className: "select-row"
+            },
+            React.createElement(Select, {
+                name: this.props.name,
+                className: "select-element",
+                options: this.props.options,
+                value: this.props.value,
+                clearable: false,
+                onChange: this.props.selectOnChange
+            }),
+            React.createElement("button", {
+                className: "select-delete",
+                onClick: this.props.deleteOnClick
+            }, "Удалить")
+        )
+    }
+
+});
+
+
+var SelectList = React.createClass(
+    {
+        displayName: "SelectList",
+
+        getInitialState: function () {
+            return {currentChoices: this.props.currentChoices}
```

Comment:
[Props in getInitialState Is an Anti-Pattern](https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html)